packages/sdk/src/interaction/interaction.js

```diff
@@ -260,7 +260,6 @@ const hardMode = ix => {
 const recPipe = async (ix, fns = []) => {
   try {
     ix = hardMode(await ix)
-    if (isBad(ix)) throw new Error(`Interaction Error: ${ix.reason}`)
```

Comment:
I am fairly certain its being used in more than the `build-at-block-height.js` so this might not be a safe removal just yet.